Packs/PrismaSaasSecurity/Integrations/SaasSecurity/SaasSecurity.py

```diff
@@ -432,6 +432,13 @@ def fetch_incidents(client: Client, first_fetch_time, fetch_limit, fetch_state,
     incidents = list()
     for inc in results:
 
+        # We fetch the incidents by the "updated-at" field,
+        # So we need to filter the incidents created before the last_fetch
+        date_created = inc.get('created_at')
+        if datetime.strptime(date_created, SAAS_SECURITY_DATE_FORMAT) < \
+                datetime.strptime(last_fetch, SAAS_SECURITY_DATE_FORMAT):
```

Comment:
Save `datetime.strptime(last_fetch, SAAS_SECURITY_DATE_FORMAT)` in a variable instead of calculating it each iteration.